luxonis_ml/tracker/tracker.py

```diff
@@ -500,6 +500,30 @@ def upload_artifact(
                 )  # Stores details for retrying later
                 self.log_stored_logs_to_mlflow()
 
+    @rank_zero_only
+    def log_matrix(
+        self, matrix: np.ndarray, name: str = "confusion_matrix"
```

Comment:
Let's remove the defualt name value so it is more general and in the docstring don't mention confusion matrix in particular, just say any matrix